### PR TITLE
Add -latomic to fix the build with NDK r11.

### PR DESCRIPTION
**Pull request by szurbrigg**

### CODE REVIEW

**Comment by JakeWharton** [comment]
lgtm

**Comment by JakeWharton** [comment]
should we update the `.travis.yml` to pull this new NDK version?

**Comment by szurbrigg** [comment]
Yeah, I'll update it.

**Comment by swankjesse** [comment]
LGTM

File: duktape/build.gradle [hunk 1]
```diff
@@ -27,7 +27,9 @@ model {
     ndk {
       moduleName = 'duktape'
       stl = "c++_static"
-
+      toolchain = "clang"
```
Comment (szurbrigg) [inline duktape/build.gradle]: c++_static is precompiled with clang.  We were using gcc which seemed to lead to badness on ARM when the two mixed (specifically the tests that caused C++ exceptions to be thrown in the JNI code).

File: .travis.yml [hunk 2]
```diff
@@ -14,10 +14,11 @@ jdk:
 before_script:
   # Install the NDK.
   - pushd ~
-  - wget http://dl.google.com/android/ndk/android-ndk-r10e-linux-x86_64.bin
-  - chmod +x android-ndk-r10e-linux-x86_64.bin
-  - ./android-ndk-r10e-linux-x86_64.bin 1> /dev/null
-  - export ANDROID_NDK_HOME="$HOME/android-ndk-r10e"
+  - wget http://dl.google.com/android/repository/android-ndk-r11b-linux-x86_64.zip
+  - unzip -qo android-ndk-r11b-linux-x86_64.zip
+  # Work around bug in NDK W.R.T. using Clang (https://github.com/android-ndk/ndk/issues/28).
```
Comment (swankjesse) [inline .travis.yml]: I’m guessing this wasn't very fun to find

**Comment by JakeWharton** [comment]
:+1:

**Pull request merged**